Makefile

```diff
@@ -86,6 +86,9 @@ CATALOG_IMG ?= $(IMAGE_TAG_BASE)-operator-catalog:$(IMAGE_TAG)
 # Image URL to use all building/pushing image targets
 IMG ?= $(IMAGE_TAG_BASE)-operator:$(IMAGE_TAG)
 
+# Image URL to use for building/pushing fencing agents image
+JOB_IMG ?= $(IMAGE_TAG_BASE)-fencing-agents:$(IMAGE_TAG)
```

Comment:
NIT: I would prefer to use `fence-agent` or `fencing-agent` instead of `fencing-agents` since this image will be used for running one fence agent by a Job.
This comment is related to all the other appearances of `fencing-agents`

Reply:
I understand, but this image actually contains all the fencing agents, so I think calling it `fance-agent` might be misleading